client/src/components/App.jsx

```diff
@@ -16,7 +23,7 @@ class App extends React.Component {
 
   getInfo() {
     $.ajax({
-      url: '/:bookId',
+      url: '/author',
```

Comment:
fyi for proxies, this needs to be a full URL, e.g. http://localhost:3002/author

Reply:
I did not know this - thanks!